addthis_block/addthis_block.info.yml

```diff
@@ -0,0 +1,11 @@
+name: AddThis Block
+description: Provides a configurable block to render AddThis elements.
```

Comment:
@nerdstein My thought was to keep some of the abstraction from the D7 version of the module. The core addthis module will act more of an API for integration with the services and providing the basic elements that will be needed. Depending on how they want to use the AddThis functionality, they can enable the block, fields or go custom. 

Working on documenting this out better in the Arch document.